addon/models/scope.js

```diff
@@ -1,9 +1,10 @@
 import Model, { attr, belongsTo, hasMany } from "@ember-data/model";
 
 export default class GroupModel extends Model {
-  @attr("string") name;
-  @attr("string") description;
+  @attr name;
+  @attr description;
```

Comment:
Why did you drop the type?

Reply:
The way i learned it is: The transforms are only needed if the json type returned from the backend does not match the expected type. otherwise the transform is not needed.

Reply:
Just read the docs again, TIL ^^